src/index.tsx

```diff
@@ -73,3 +79,17 @@ export function debug() {
 export function didHandleURL(url: string): Promise<boolean> {
   return AppcuesReactNative.didHandleURL(url);
 }
+
+const ComponentName = 'AppcuesFrameView';
+
+type AppcuesFrameProps = {
+  frameID: string;
+  style: ViewStyle;
```

Comment:
@mmaatttt None of the docs or examples show `style` as a required prop, but this TypeScript definition forces it to be defined. Was this intended to be optional? 

```diff
- style: ViewStyle;
+ style?: ViewStyle; 
```

Reply:
@ashfurrow great question, thanks for digging into the code. You're correct that `style` being optional would make sense.

I can update this and push out a patch release for you.

Reply:
You can grab version 3.1.5 with the type fix.